react-app/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Introducing RTL tests from day 1 is good, though the test will fail if you remove default part of your App's JSX. You may try to refactor that to still pass and test for your actual components existence on the page.